manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml

```diff
@@ -403,19 +401,6 @@ spec:
         path: collection.logs.fluentd.nodeSelector
         x-descriptors:
         - 'urn:alm:descriptor:com.tectonic.ui:nodeSelector'
-      - description: Resource requirements for the Curator pods
```

Comment:
@alanconway I think this has to remain for compatibility reasons but is just no longer honored.  Maybe we just need to update the main "curation" note